vrs/utils/PixelFrame.h

```diff
@@ -30,6 +30,7 @@
 #include <vrs/RecordReaders.h>
 #include <vrs/utils/DecoderFactory.h>
 #include <vrs/utils/VideoFrameHandler.h>
+#include <jpeglib.h>
```

Comment:
We ran into some header collisions with third party libraries, with this class even, so we try hard to avoid polluting includes, such as this one. In this case, you can make a local function that does this while passing the `PixelFrame&` to do the work.

Reply:
@georges-berenger since `readJpegFrameHelper` has to access the private variables `frameBytes_` and `imageSpec_` of a `PixelFrame`, is it even possible to refactor into a static local function? (my C++ is very rusty, so happy to learn if there actually is a way)

Reply:
alternatively, we could just not use a helper function at all and duplicate the common logic in both the `readJpegFrameFromFile` and `readJpegFrame` functions. any preferences for the approach?

Reply:
What I suggested should be possible, I checked all the parameters used. So unless you try and find something I have seen, please do proceed that way.

Reply:
@georges-berenger  you're right, I missed the getters :) pushed a new commit that should address all your comments!